components/layout_2020/formatting_contexts.rs

```diff
@@ -59,7 +59,7 @@ pub(crate) enum NonReplacedFormattingContextContents {
     // Other layout modes go here
 }
 
-/// The baselines of a layout or a [`BoxFragment`]. Some layout uses the first and some layout uses
+/// The baselines of a layout or [`Fragment::Box`] Some layout uses the first and some layout uses
```

Comment:
Again, I think this just needs to be qualified. `Fragment::Box` is an enum variant, but this is actually referring to the struct stored inside the enum variant `BoxFragment`.